nav2_bringup/bringup/launch/bringup_launch.py

```diff
@@ -37,7 +37,7 @@ def generate_launch_description():
     map_yaml_file = LaunchConfiguration('map')
     use_sim_time = LaunchConfiguration('use_sim_time')
     params_file = LaunchConfiguration('params_file')
-    bt_xml_file = LaunchConfiguration('bt_xml_file')
+    default_bt_xml_filename = LaunchConfiguration('default_bt_xml_filename')
```

Comment:
Add to the migration guide https://github.com/ros-planning/navigation.ros.org/blob/master/migration/Eloquent.rst that this variable changed for the launch system & in the param file. Add a new section with the new capability that you can select  specific request BTs / change in the action message definiton.

Reply:
done